src/mesh/exodusII_io_helper.C

```diff
@@ -1663,7 +1663,25 @@ void ExodusII_IO_Helper::initialize(std::string str_title, const MeshBase & mesh
     for (const auto & id : shellface_boundaries)
       unique_side_boundaries.push_back(id);
   }
+  // Add any empty-but-named side boundary ids
+  for (const auto & pair : bi.get_sideset_name_map())
+    {
+      const boundary_id_type id = pair.first;
+
+      if (std::find(unique_side_boundaries.begin(),unique_side_boundaries.end(),id)
+            == unique_side_boundaries.end())
+        unique_side_boundaries.push_back(id);
+    }
+
   bi.build_node_boundary_ids(unique_node_boundaries);
+  for (const auto & pair : bi.get_nodeset_name_map())
+    {
+      const boundary_id_type id = pair.first;
+
+      if (std::find(unique_node_boundaries.begin(),unique_node_boundaries.end(),id)
```

Comment:
Usually we put a space after commas, right? Maybe this is not an official coding style guideline.

Reply:
We do, and it's not but it should be.  Will fix.